R/render.R

```diff
@@ -28,10 +32,29 @@ render <- function(data,
 
 
 
+#' `r lifecycle::badge("experimental")`
+#' @title Render a data.frame or tibble specific to producing tableone.
+#'
+#' @description Render a previously created data.frame to html, rtf or latex
+#'
+#' @param data Input data.frame or tibble to visualize
```

Comment:
could you use @inheritparams render?

Reply:
Looking at the documented functions again - I agree. This is not efficient. I'll propose a simplier solution using inherits, etc. I think the issue is its missing a default function....